__tests__/app.test.js

```diff
@@ -132,5 +127,40 @@ describe("ARTICLES", () => {
                 });
             });
         });
+
+        describe("/api/articles/:article_id/comments", () => {
+            describe("Status 200", () => {
+                test("responds with an array of all comments for the given article sorted by date with the most recent comments first", () => {
+                    return request(app)
+                    .get("/api/articles/5/comments")
+                    .expect(200)
+                    .then(({body}) => {
+                        const commentsWithArticleId5 = body.comments
+                        expect(Array.isArray(commentsWithArticleId5)).toBe(true)
+                        expect(commentsWithArticleId5).toBeSortedBy("created_at", {descending : true})
+                        expect(commentsWithArticleId5).toMatchObject(
+                            [
+                            {
+                            comment_id: expect.any(Number),
+                            votes: expect.any(Number),
+                            created_at: expect.any(String),
+                            author: expect.any(String),
+                            body: expect.any(String),
+                            article_id: expect.any(Number)
+                        },
+                        {
+                            comment_id: expect.any(Number),
+                            votes: expect.any(Number),
+                            created_at: expect.any(String),
+                            author: expect.any(String),
+                            body: expect.any(String),
+                            article_id: expect.any(Number)
+                        }
+                    ]
+                    )
+                    })
+                })
+            })
```

Comment:
3 tests missing here:
Status 200, valid ID, but has no comments responds with an empty array of comments
Status 400, invalid ID, e.g. string of "not-an-id"
Status 404, non existent ID, e.g. 0 or 9999